Documentation/AEPEdgeIdentity.md

```diff
@@ -0,0 +1,137 @@
+# Getting Started
```

Comment:
nit: can we rename this file to something like `getting-started-test-app.md` to be in line with the other files to be added in this folder?

Reply:
Renamed file to suggested name!